src/gui/transferlistwidget.cpp

```diff
@@ -939,12 +970,18 @@ void TransferListWidget::displayListMenu(const QPoint &)
         if (torrent->hasMetadata())
             needsPreview = true;
 
+        if (!oneVersion1 && torrent->infoHash().v1().isValid())
+            oneVersion1 = true;
```

Comment:
It is also possible to write it `oneVersion1 |= torrent->infoHash().v1().isValid();` what do you think?

Reply:
>what do you think?

I don't think it's better. It will perform assignment (i.e. memory write) each time whereas current code does this only once. Also, I don't like using bitwise operations with boolean type.